examples/lessons/11/app.js

```diff
@@ -0,0 +1,206 @@
+/* eslint-disable no-var, max-statements */
```

Comment:
Just a thought: Is linting set up for the examples? Long term we want to remove as many of these linter exceptions as possible. Especially the `no-var` would be good to get rid off. Try to remove the others and do `npm run lint` to see if they are still needed.

Reply:
Lint still doesn't work for me, now I manually make sure there is no vars in my example code, so shouldn't keep these comments here.